mantidproject/mantid
# Add mantidpython launch script to MantidPlot application directory on OSX

Closes #13728 

Added a `mantidpython` script to the `MantidPlot.app/Contents/MacOS/`. This is equivalent to the script of the same name in the `bin` directory of linux installations of Mantid. It can be used to launch python using Mantid's python environment.

I have also added launcher applications for ipython console and ipython notebooks. These can optionally be installed from the MantidPlot dmg image. As they are applications they can be added to the dock etc.

**To Test:**

Must be tested on OS X.

I've put the dmg file from the build server at `smb://olympic/babylon5/Scratch/Matthew Jones` (commit ed1ac08).

Please check that the launchers work. The `mantidpython` script is used by the launchers, so testing the launchers also tests the script.
